legal-api/src/legal_api/models/filing.py

```diff
@@ -238,13 +235,27 @@ class Source(Enum):
                 }
             }
         },
+        'specialResolution': {'name': 'specialResolution', 'title': 'Special Resolution',
+                              'codes': {
+                                  'CP': 'SPRLN'}},
+        'transition': {
+            'name': 'transition',
+            'title': 'Transition',
+            'codes': {
+                'BC': 'TRANS',
+                'BEN': 'TRANS',
+                'ULC': 'TRANS',
+                'CC': 'TRANS'
+            }
+        },
+
         # changing the structure of fee code in courtOrder/registrarsNotation/registrarsOrder
         # for all the business the fee code remain same as NOFEE (Staff)
+        'adminFreeze': {'name': 'adminFreeze', 'title': 'Admin Freeze', 'code': 'NOFEE'},
         'courtOrder': {'name': 'courtOrder', 'title': 'Court Order', 'code': 'NOFEE'},
-        'registrarsNotation': {'name': 'registrarsNotation', 'title': 'Registrars Notation', 'code': 'NOFEE'},
-        'registrarsOrder': {'name': 'registrarsOrder', 'title': 'Registrars Order', 'code': 'NOFEE'},
         'putBackOn': {'name': 'putBackOn', 'title': 'Put Back On', 'code': 'NOFEE'},
-        'adminFreeze': {'name': 'adminFreeze', 'title': 'Admin Freeze', 'code': 'NOFEE'}
+        'registrarsNotation': {'name': 'registrarsNotation', 'title': 'Registrars Notation', 'code': 'NOFEE'},
+        'registrarsOrder': {'name': 'registrarsOrder', 'title': 'Registrars Order', 'code': 'NOFEE'}
```

Comment:
Why are these filings compressed to one liners, but the others have added spacing?

Reply:
Each of these are less than 120 characters (which is the max-line-length configuration in lint). Kept it separate from alphabetical order since these are staff only filings

Reply:
I was confused because some of the expanded filings are less than 120 characters, for example:
`'conversion': {'name': 'conversion', 'title': 'Conversion Ledger', 'codes': {'SP': 'FMCONV', 'GP': 'FMCONV'}, },`

Reply:
The basic principle is to make sure it's readable. It's not mandatory to keep it in one line if its < 120 characters. `conversion` has nested object, which is more readable in multi-line than single line.